packages/ember-testing/lib/test/adapter.js

```diff
@@ -28,6 +28,4 @@ export function asyncEnd() {
 
 function adapterDispatch(error) {
   adapter.exception(error);
-  
-  console.error(error.stack);// eslint-disable-line no-console 
```

Comment:
Yep, we need to bring this back (and explicitly mock in tests where we expect an error).

Specifically, this console.error is the _only_ thing that notifies the console of failures (including the stack trace) when unexpected exceptions or rejections happen in tests...

Reply:
Something like this?

```javascript
function adapterDispatch(error) {
   adapter.exception(error);
   
   adapter.outputError(error.stack); 
```

`outputError` is by default `console.error` and can be overridden with a noop like this.

```
Test.adapter = QUnitAdapter.create({
   outputError () {}
});
```

Reply:
I was more thinking to just override `console.error` in the specific tests, then reset it in the `afterEach`